src/js/media-poster-image.js

```diff
@@ -0,0 +1,70 @@
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+import { MediaUIAttributes } from './constants.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+  <style>
+    :host {
+      height: 100%;
+      width: 100%;
+      position: relative;
+    }
+
+    div {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-position: center;
+      background-size: cover;
+      background-repeat: no-repeat;
+    }
+
+    :host([${MediaUIAttributes.MEDIA_HAS_PLAYED}]) div {
+      display: none;
+    }
+  </style>
+
+  <div part="placeholder"></div>
+  <div part="image"></div>
```

Comment:
If we're gonna use parts here, just understand they become part of the API, with all the downsides harder changes in the future.

Reply:
I'm totally fine with swapping these out for IDs. My reasoning was that it could be nice to make it easier to target these for things like specifying a different `background-size` or adding a blur to the placeholder. I'm totally fine with keeping this as simple as possible and just pointing people towards using a div if they want to customize things. Thoughts?